scripts/lib/immutable-eval-corpus.js

```
@@ -0,0 +1,290 @@
+    failures.push('trial.invalid');
+  }
+  if (!Number.isFinite(evidence.metrics?.durationMs) || evidence.metrics.durationMs < 0) {
+    failures.push('metrics.duration');
+  }
+  if (!Number.isFinite(evidence.metrics?.tokensUsed) || evidence.metrics.tokensUsed < 0) {
+    failures.push('metrics.tokens');
+  }
+  return failures;
+}
+
+function evaluateCase({ packet, allPackets, manifest, evidence, expectedBinding }) {
```

ESLint

[error] Refactor this function to reduce its Cognitive Complexity from 17 to the 15 allowed.

(sonarjs/cognitive-complexity)